lib/house.rb

```diff
@@ -0,0 +1,58 @@
+class House
+  attr_reader :price, :address, :rooms
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def turn_price_to_int
+    @price_int = @price[1..-1].to_i
```

Comment:
We generally want to try to avoid destroying any data - what if in the future we want the string version of the price?  It would be better if this method simply returned the integer version of the price instead of changing the instance variable.